tensorflow/python/keras/applications/imagenet_utils.py

```diff
@@ -66,11 +66,11 @@
       {ret}
 
   Raises:
-      ValueError: In case of unknown `data_format` argument.
+      ValueError: In case of unknown `mode` or `data_format` argument.
```

Comment:
This cannot be added here as this doc is shared across all applications. This will have to be made a placeholder similar to {mode} / {ret} above.

Reply:
I was thinking that the `mode`  value error should be in the comments across all variations because it isn't something specific to a certain mode, just like the `data_format` value error. From what I see, the {mode} placeholder exists because a different docstring is needed for each mode. Is this still the case? (Please let me know if I'm missing something, I'm probably not understanding a detail here.)

Reply:
Please checkout how mobilenet reuses this docstring. `mode` is not an input in the `preprocess_input` function of mobilenet.

Reply:
Thanks for the pointer. I made changes to the docstring based on your comment, specifically by adding a placeholder and reflecting this change across all models. Please take a look at let me know what you think!